<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditTor</class>
 <widget class="QWidget" name="EditTor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EditTor</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="user_l">
     <property name="text">
      <string>Custom Path</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="executable_path"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="password_l">
     <property name="text">
      <string>Extra Args</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="extra_args"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="port_range_l">
     <property name="text">
      <string>Tor Config</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QTextEdit" name="torrc"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout"/>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="multiplexing_l">
     <property name="text">
      <string>Data DIr</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="data_directory"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
